test/gtest/shp/shp-tests.cpp

```diff
@@ -4,25 +4,37 @@
 
 #include "shp-tests.hpp"
 
-using AllTypes = ::testing::Types<dr::shp::distributed_vector<int>,
-                                  dr::shp::distributed_vector<float>>;
+// Use this for shorter build time
+#define MINIMAL_TEST 1
+#ifdef MINIMAL_TEST
+
+using AllTypes = ::testing::Types<xhp::distributed_vector<int>>;
+#include "common/zip.hpp"
+
+#else
+
+using AllTypes = ::testing::Types<xhp::distributed_vector<int>,
+                                  xhp::distributed_vector<float>>;
 
 #include "common/all.hpp"
+#include "common/distributed_vector.hpp"
 // Issue with 2 element zips
```

Comment:
What is this issue? Does shp::enumerate / shp::zip does not work for two elements? If so I'd like to report a bug for it to keep it in mind. If it is fixed let's remove the comment.

Reply:
This PR fixes the issue. I will remove the enumerate comment before merging.